pkgs/applications/networking/instant-messengers/telegram/telegram-qml/default.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenv, fetchFromGitHub
+, qt5Libs, qtbase, qtmultimedia, qtquick1
+, libqtelegram-aseman-edition }:
+
+stdenv.mkDerivation rec {
+  name = "telegram-qml-${version}";
+  version = "v0.9.1-stable";
+
+  src = fetchFromGitHub {
+    owner = "Aseman-Land";
+    repo = "TelegramQML";
+    rev = version;
+    sha256 = "077j06lfr6qccqv664hn0ln023xlh5cfm50kapjc2inapxj2yqmn";
+  };
+
+  buildInputs = [ qtbase qtmultimedia qtquick1 libqtelegram-aseman-edition ];
+  enableParallelBuild = true;
+
+  patchPhase = ''
+    substituteInPlace telegramqml.pro --replace "/\$\$LIB_PATH" ""
+    substituteInPlace telegramqml.pro --replace "INSTALL_HEADERS_PREFIX/telegramqml" "INSTALL_HEADERS_PREFIX"
+  '';
+
+  configurePhase = ''
+    qmake -r PREFIX=$out BUILD_MODE+=lib
+  '';
+
+  meta = {
+    description = "Telegram API tools for QtQml and Qml. It's based on Cutegram-Core and libqtelegram.";
```

Comment:
Please avoid pkg name and dots ion the description. Read rules https://nixos.org/nixpkgs/manual/#sec-standard-meta-attributes